TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+### Remove mobile activity panel toggle #6539
+
+1. Narrow your viewport to < 782px.
+2. Navigate to various WooCommerce pages.
+3. Make sure the various tabs can be seen and function as expected.
+4. Open the "Inbox" panel.
```

Comment:
You _might_ mention that the Inbox Panel isn't available on the Homescreen. Or perhaps I'm just really trying to find something to say on this PR.

Reply:
Added 😄